client/src/reducers/index.js

```diff
@@ -0,0 +1,5 @@
+const mentorQ = (state: [], action) => {
+	return state;
```

Comment:
**You haven't implemented any actions yet. Implement a simple action that's just passing text from a react component to the store for example.**